BritneyS/SFWCardsAgainstHumanity
# Refactor 2

## What you did :question:
- Refactor: removed unused elements
- Refined error messages
- No app behavioral change

## How you did it :white_check_mark:
- Removed unused classes
- Removed unused variable isLoading
- Removed unnecessary method viewDidLoad in HomeViewController
- Removed unnecessary print statements (mostly in successful/pass clauses
- Added 'error' to print statements in fail clauses for clearer error message printing to terminal


## How to test it :microscope:
- Run the app
- Tap 'Play!'
- Choose any selection
- Tap 'See Selection'
- Tap the favorite icon (heart) once to add the visible selection the Favorites (heart should be filled)
- Tap 'X' to dismiss Selection View
- Tap 'Back' to return to the Home View
- Tap 'Faves' to navigate to Favorites
- Note the saved favorite
- Tap the black card label that represents the saved favorite
- Tap the favorite icon (heart) to delete the visible selection from Favorites (heart should be empty)
- Tap 'X' to dismiss Selection View
- Deleted favorite should no longer be visible in the collection view


## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-10-28 at 11 20 05](https://user-images.githubusercontent.com/8409475/47618039-bf133980-daa4-11e8-94e4-09cb05dd2b44.png)
![simulator screen shot - iphone xr - 2018-10-28 at 11 20 15](https://user-images.githubusercontent.com/8409475/47618042-c5091a80-daa4-11e8-9286-1198245aee3e.png)
![simulator screen shot - iphone xr - 2018-10-28 at 11 20 22](https://user-images.githubusercontent.com/8409475/47618047-cb979200-daa4-11e8-86a2-5e60f290985d.png)
![simulator screen shot - iphone xr - 2018-10-28 at 11 20 30](https://user-images.githubusercontent.com/8409475/47618050-d18d7300-daa4-11e8-930d-e2fd186e919c.png)
![simulator screen shot - iphone xr - 2018-10-28 at 11 20 37](https://user-images.githubusercontent.com/8409475/47618054-d7835400-daa4-11e8-8700-7cb526550b4f.png)
![simulator screen shot - iphone xr - 2018-10-28 at 11 20 42](https://user-images.githubusercontent.com/8409475/47618055-e0742580-daa4-11e8-8155-7d539c1833ff.png)
